src/LaravelRequestDocs.php

```diff
@@ -119,15 +119,23 @@ public function filterByMethods($docs, $get, $post, $put, $path, $delete, $head)
 
     public function groupDocs($docs, $group = 'default')
     {
+        if ($group === 'default') {
+            return $docs;
+        }
+
+        $groupDocs = [];
+
         if ($group === 'api_uri') {
-            return $this->groupDocsByAPIURI($docs);
+            $groupDocs = $this->groupDocsByAPIURI($docs);
         }
 
         if ($group === 'controller_full_path') {
-            return $this->groupDocsByFQController($docs);
+            $groupDocs = $this->groupDocsByFQController($docs);
         }
 
-        return $docs;
+        return collect($groupDocs)->sortBy(['group', 'group_index'])
```

Comment:
Probably this

```

        return collect($groupDocs)->groupBy(['group', 'group_index'])
            ->values()
            ->flatten(2)
            ->toArray();
```

Reply:
```
        $groupBy = ['group', 'group_index'];
        return collect($groupDocs)->groupBy($groupBy)
            ->values()
            ->flatten(count($groupBy))
            ->toArray();
```

Reply:
Fixes here #150

Reply:
ah UT failed, so will revert changes https://github.com/rakutentech/laravel-request-docs/pull/150/commits/c93806366822cfaee6f4b1d9f99d78d99056f84e 
Will wait for your pull req